pkgs/development/libraries/polkit-qt-1/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, mkDerivation
+, fetchurl
+, cmake
+, pkg-config
+, polkit
+, glib
+, pcre
+, libselinux
+, libsepol
+, utillinux
```

```suggestion
, util-linux
```
I think ofborg wants you to use the hyphenated name?

---

ofborg and hydra do not allow using aliases and utillinux is an alias to util-linux.